test/integration/targets/win_package/defaults/main.yml

```diff
@@ -2,6 +2,8 @@
 # spaces are tricky, let's have one by default
 test_win_package_path_safe: C:\ansible\win_package
 test_win_package_path: C:\ansible\win package
+test_win_package_log_path_install: C:\ansible\win_package\test-install.log
```

Comment:
I would use the unsafe path variable just to make sure we are escaping spaces correctly, e.g. `test_win_package_log_path_install: '{{ test_win_package_path }}\test-install.log`. The same applies to the uninstall log.

Reply:
good point ... done